src/fsharp/FSharp.Core.Unittests/SurfaceArea.coreclr.fs

```diff
@@ -293,6 +293,222 @@ Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityC
 Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityComparer`1[T] Structural[T]()
 Microsoft.FSharp.Collections.HashIdentity: System.String ToString()
 Microsoft.FSharp.Collections.HashIdentity: System.Type GetType()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Boolean Equals(System.Object)
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Int32 GetHashCode()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: System.String ToString()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: System.Type GetType()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Void ChainComplete(Microsoft.FSharp.Core.Unit ByRef, Int32)
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Void ChainDispose(Microsoft.FSharp.Core.Unit ByRef)
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: Boolean Equals(System.Object)
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: Boolean ProcessNext(T)
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: Int32 GetHashCode()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: System.String ToString()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: System.Type GetType()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: Void .ctor()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity`2[T,TResult]: Void ChainComplete(Microsoft.FSharp.Core.Unit ByRef, Int32)
```

Comment:
This was once again an optimization to minimize the small sized enumerations. Maybe adding a compiler attribute so that it knows not to insert tail instruction?